src/components/PayBill/Paybill.test.jsx

```diff
@@ -0,0 +1,12 @@
+import PayBill from "./PayBill";
+import React from "react";
+
+describe("PayBill", () => {
+  describe("render", () => {
+    describe("snapshots 📸", () => {
```

Comment:
maybe we can show different snaps when `billName` and `amountRemainingToPay` props are defined

Reply:
Sure, I can add that in a bit